### PR TITLE
Logger and correct some fix

### PR DESCRIPTION
**Pull request by TheYisusGithub**

### CODE REVIEW

File: log4j.properties [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+log4j.appender.FileAppender.File=log/log.log
+log4j.appender.HTML.File=./log/application.html
+
+# Here we define the layout and pattern
+log4j.appender.FileAppender.DatePattern='.'MM-dd-yyyy
+log4j.appender.FileAppender.layout=org.apache.log4j.PatternLayout
+log4j.appender.FileAppender.layout.ConversionPattern= %d{HH:mm:ss} %-5p [%C{1}][%M][%L] %m%n
+log4j.appender.HTML.layout=org.apache.log4j.HTMLLayout
+log4j.appender.HTML.layout.Title=Application log
+log4j.appender.HTML.layout.LocationInfo=true
```
Comment (MadayAlcala) [inline log4j.properties]: Please add a extra line.

File: src/test/java/kanbanflow/runner/Runner.java [hunk 2]
```diff
@@ -59,7 +60,8 @@ public void screenShot(ITestResult result) {
                 System.out.println("Exception while taking screenshot " + e.getMessage());
             }
         }
-        driver.quit();
+//        driver.quit();
```
Comment (MadayAlcala) [inline src/test/java/kanbanflow/runner/Runner.java]: Please  delete unnecessary comments.

Comment (MadayAlcala) [reply src/test/java/kanbanflow/runner/Runner.java]: And delete unnecesary line.

File: src/test/java/kanbanflow/runner/Runner.java [hunk 3]
```diff
@@ -59,7 +60,8 @@ public void screenShot(ITestResult result) {
                 System.out.println("Exception while taking screenshot " + e.getMessage());
```
Comment (MadayAlcala) [inline src/test/java/kanbanflow/runner/Runner.java]: Please change the sout.

File: src/test/java/kanbanflow/runner/Runner.java [hunk 4]
```diff
@@ -68,5 +70,7 @@ public void screenShot(ITestResult result) {
     @AfterTest
     public void afterExecution() {
         ReportGenerator.generateReport();
+
```
Comment (MadayAlcala) [inline src/test/java/kanbanflow/runner/Runner.java]: Please delete unnecessary line.

File: src/test/java/kanbanflow/runner/Runner.java [hunk 5]
```diff
@@ -68,5 +70,7 @@ public void screenShot(ITestResult result) {
...
     }
+
```
Comment (MadayAlcala) [inline src/test/java/kanbanflow/runner/Runner.java]: Please delete unnecessary line.

File: src/test/resources/features/InviteMemberUser.feature [hunk 6]
```diff
@@ -7,7 +7,7 @@ Feature: Creation funtional and acceptance
   Then The user clicking should show the 30testemail@gmail.com with the pending invitations
```
Comment (MadayAlcala) [inline src/test/resources/features/InviteMemberUser.feature]: Please delete unnecessary line.

File: log4j.properties (hunk 1 above)
Comment (TheYisusGithub) [reply log4j.properties]: Done.

File: src/test/java/kanbanflow/runner/Runner.java (hunk 2 above)
Comment (TheYisusGithub) [reply src/test/java/kanbanflow/runner/Runner.java]: Done.

File: src/test/java/kanbanflow/runner/Runner.java (hunk 3 above)
Comment (TheYisusGithub) [reply src/test/java/kanbanflow/runner/Runner.java]: Done.

File: src/test/java/kanbanflow/runner/Runner.java (hunk 4 above)
Comment (TheYisusGithub) [reply src/test/java/kanbanflow/runner/Runner.java]: Done.

File: src/test/java/kanbanflow/runner/Runner.java (hunk 5 above)
Comment (TheYisusGithub) [reply src/test/java/kanbanflow/runner/Runner.java]: Done.

File: src/test/resources/features/InviteMemberUser.feature (hunk 6 above)
Comment (TheYisusGithub) [reply src/test/resources/features/InviteMemberUser.feature]: Done.

**Review by MadayAlcala — APPROVED** [review]

**Pull request merged**